### ISSUE TITLE
Word echo does not work with VoiceOver under MacOs

### ISSUE THREAD

**Issue by andadapt** [microsoft/vscode#112171]
Under MacOS words typed into the VSCODE editor are not echoed.

Character echo works fine.

To replicate this issue. Open a new file. Type in a word and press space.

The characters of the word will be echoed but the whole word will not.

**Comment by isidorn** [comment]
@andadapt I acknowledge this issue that I can reproduce with VS Code stable.
However the latest Electron 11 version seems to fix this. We plan to ship VS Code stable with Electron 11 at end of January so then this should be fixed.

However it would be cool if you try out the exploration build and confirm to us that it works fine for you
https://az764295.vo.msecnd.net/exploration/d530b3b27d0f52a0290cdb53031ba60f02927c60/VSCode-darwin.zip

Thanks

**Comment by andadapt** [comment]
I can confirm this build works. Echo is working for character and word

**Comment by isidorn** [comment]
@andadapt thanks for letting us know
